v2/datastream-common/src/main/java/com/google/cloud/teleport/v2/datastream/sources/DataStreamIO.java

```diff
@@ -172,6 +174,23 @@ public DataStreamIO withDirectoryWatchDuration(Duration directoryWatchDuration)
     return this;
   }
 
+  /**
+   * Set applyReshuffle to false in order to skip the reshuffle step on Datastream records.
+   *
+   * <p>In the Datastream to Spanner template, one reshuffle step is already performed to balance
```

Comment:
Nit: DatastreamIO is generic we shouldn't reference a specific use-case for it. Just add that this to provide an additional flexibility in terms of how DatastreamIO outputs the records.